db/migrate/20171125173941_create_order.rb

```diff
@@ -1,7 +1,6 @@
 class CreateOrder < ActiveRecord::Migration[4.2]
   def change
-    create_table :orders do |t|
-      t.timestamps
+    create_table :orders do
```

Comment:
If you are not adding any attributes, then don't include a block. Remove the `do` and  `end`

Reply:
What I mean is that 
```
create_table :orders do
end 
```
Should be 
```
create_table :orders
```